test/lit.cfg

```diff
@@ -263,9 +263,11 @@ config.substitutions.append( ('%test_debuginfo', ' ' + config.llvm_src_root + '/
 config.substitutions.append( ('%itanium_abi_triple', makeItaniumABITriple(config.target_triple)) )
 config.substitutions.append( ('%ms_abi_triple', makeMSABITriple(config.target_triple)) )
 
-config.substitutions.append( ('%flang1', ' ' + config.flang + '1 ') )
-config.substitutions.append( ('%flang2', ' ' + config.flang + '2 ') )
-config.substitutions.append( ('%flang', ' ' + config.flang + ' ') )
+flang_root = os.path.splitext(config.flang)[0]
+flang_suffix = os.path.splitext(config.flang)[1]
```

Comment:
Just for the sake of completeness I should ask, how does it work on the systems where executable files have no suffixes? (namely, `flang2` instead of `flang2.exe`)

Reply:
If there's no suffix, then `flang_suffix` = '' (empty string)